static/js/settings_org.js

```diff
@@ -824,19 +849,6 @@ exports.build_page = function () {
         e.stopPropagation();
     });
 
-
-    $('#admin_auth_methods_table').change(function () {
```

Comment:
There are some occurrences of `simulate_auth_methods` in settings_org node tests as well, make sure to remove/update them as well.